src/ducks/alerts.js

```diff
@@ -16,8 +16,8 @@ export function reducer(previous_state = [], action) {
       }
 
       notification.error({
-        message: action.error.name,
-        description
+        message: 'Password requirements were not met.',
```

Comment:
I think this shouldn't hard coded, cause there might be some other login errors.